config/300-crds/300-stepaction.yaml

```diff
@@ -1113,3 +1113,11 @@ spec:
       - tekton
       - tekton-pipelines
   scope: Namespaced
+  conversion:
```

Comment:
is it possible to add any tests for this ?
cc @afrittoli

Reply:
When I checked the conversion reconciler [logic](https://github.com/knative/pkg/blob/b7bbf4be5dbd9a8b8b60e06094bd8930cf241357/webhook/resourcesemantics/conversion/reconciler.go#L98-L103), the only idea I can think of right now is to add a test in the [e2e/conversion_test](https://github.com/tektoncd/pipeline/blob/main/test/conversion_test.go) to verify whether the CRD's Strategy is set to Webhook. WDYT?

Reply:
@waveywaves Could you check https://github.com/tektoncd/pipeline/pull/8644/commits/d5944bfe51ed5cf974e86a12de57fe7358d6e3b0 ? Thx in advance :)

Reply:
I think the test looks good for now based on how the reconciler in knative is checking for the webhook strategy